test/serializers/associations_test.rb

```diff
@@ -125,6 +125,27 @@ def test_associations_custom_keys
         assert expected_association_keys.include? :site
       end
 
+      class BelongsToTestPostSerializer < ActiveModel::Serializer
+        belongs_to :blog
+      end
+
+      def test_belongs_to_doesnt_load_not_included_record_for_jsonapi
+        post = Post.new(blog_id: 'the_blog_id')
+        def post.blog
+          fail 'should use blog_id'
+        end
+
+        def post.read_attribute_for_serialization(name)
+          # Bug in how ActiveModelSerializers::Model handles attributes and methods
+          send(name)
```

Comment:
does that comment saying you're implementing the bug, or avoiding it? I'm not clear on that one.